adserver/templates/adserver/email/flight_wrapup.html

```diff
@@ -0,0 +1,24 @@
+{% extends "adserver/email/base.html %}
+{% load i18n %}
+
+
+<p>{% blocktrans %}Hi,{% endblocktrans %}</p>
+
+<p>{% blocktrans with site_name=site.name %}Thank you for advertising with {{ site_name }}!{% endblocktrans %}</p>
+
+<p>{% blocktrans %}Your campaign haa wrapped up.{% endblocktrans %}</p>
```

Comment:
```suggestion
<p>{% blocktrans %}Your campaign has wrapped up:{% endblocktrans %}</p>
```

Reply:
I have a followup commit that fixes this.